server/api/models/recipe.js

```diff
@@ -0,0 +1,46 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+
+const schema = new Schema(
+  {
+    servings: {
+      type: Number,
+      required: true
+    },
+    prepTime: String,
+    cookTime: String,
+    ingredients: [{
+      measurement: {
```

Comment:
Should this be `unit` instead of `measurement`? Thoughts anyone?

Reply:
I like `unit`